src/DotNet.Status.Web/Startup.cs

```diff
@@ -185,7 +185,7 @@ private void AddServices(IServiceCollection services)
                 .AddCookie(IdentityConstants.ApplicationScheme,
                     o =>
                     {
-                        o.ExpireTimeSpan = TimeSpan.FromDays(7);
+                        o.ExpireTimeSpan = TimeSpan.FromMinutes(30);
```

Comment:
This is a annoying experience, to have to log in every 30 minutes. But since the site is used so rarely, it's probably fine.

Reply:
This is only effectively "you have to log in if you haven't touched the site in 30 minutes" so its not horrible, just annoying

Reply:
Unluckily security wants everything to logout automatically and 30 minutes is maximum allowed limit :(